src/battle_script_commands.c

```diff
@@ -3444,7 +3446,7 @@ static void Cmd_getexp(void)
                     gBattleMons[0].spAttack = GetMonData(&gPlayerParty[gBattleStruct->expGetterMonId], MON_DATA_SPATK);
                     gBattleMons[0].spDefense = GetMonData(&gPlayerParty[gBattleStruct->expGetterMonId], MON_DATA_SPDEF);
                 }
-
+                // What is else if?
```

Comment:
Unnecessary

Reply:
I'll delete this from the pokefirered repo too then.